test/visibility/only/innerModuleSharesName-topLevel-useInDiffScope.good

```diff
@@ -1,2 +1,6 @@
 innerModuleSharesName-topLevel-useInDiffScope.chpl:9: warning: an implicit module named 'innerModuleSharesName-topLevel-useInDiffScope' is being introduced to contain file-scope code
+innerModuleSharesName-topLevel-useInDiffScope.chpl:11: warning: potentially surprising shadowing for 'M'
+innerModuleSharesName-topLevel-useInDiffScope.chpl:10: note: it refers to a symbol found through the 'use' statement here
+innerModuleSharesName-topLevel-useInDiffScope.chpl:2: note: leading to 'M' defined here
+innerModuleSharesName-topLevel-useInDiffScope.chpl:1: note: but, there is a shadowed symbol 'M' defined here
```

Comment:
This test is weird, because the `M` that is conflicting isn't found through the `use` statement at all. We can write e.g.

``` chapel
module Outer {
  module X { var i: int; } 
  proc main() {
    writeln(X.i); // OK, even though there was no use/import
  }
}
```

So the conflict here is between something explicitly named in a `use` statement and something available in an outer scope. But, I don't think I want the warning to apply to something explicitly named in a `use` statement in any case, so I've tried that, and it seems to remove this case without changing the behavior of the new tests I've added.